lib/plugins/aws/invoke-local/index.js

```diff
@@ -460,7 +460,13 @@ class AwsInvokeLocal {
     // Java zip/jar implementaion doesn't preserve file permissions in a zip file.
     // https://bugs.openjdk.java.net/browse/JDK-6194856
     // So we follow Java's way in java.util.zip.ZipEntry#isDirectory() to detect directory.
-    await decompress(artifact, destination, { filter: (file) => !file.path.endsWith('/') });
+    const artifactFiles = await decompress(artifact, destination, { filter: (file) => !file.path.endsWith('/') });
+    for (const file of artifactFiles) {
+      // We need to make sure there is executable permission for rust builds
+      if (file.path === 'bootstrap') {
+        await fse.chmodSync(destination + '/' + file.path, "755");
```

Comment:
Let's not use sync function along with `await`

Reply:
Let's construct the path to file in a OS-agnostic way - not all systems use `/` as path separator

Reply:
Please check https://github.com/serverless/serverless/commit/2a9e6b07ca4d96cf9e26dd24af4b02df23eb8f74